sources/web/datalab/polymer/modules/ApiManager.ts

```diff
@@ -421,17 +437,34 @@ class ApiManager {
     const successCode = options.successCode || 200;
     const request = new XMLHttpRequest();
     const noCache = options.noCache || false;
+    const failureCodes = options.failureCodes;
 
     return new Promise((resolve, reject) => {
       request.onreadystatechange = () => {
         if (request.readyState === 4) {
           if (request.status === successCode) {
+
+            // If this is the first success after failures, call the connected handler
```

Comment:
Nothing in the Datalab front-end is Chrome-only. Looks like fetch is supported by all evergreen browsers except IE. Do you think it's worth looking for a polyfill?